src/main/java/io/cdap/e2e/utils/StorageClient.java

```diff
@@ -87,10 +95,18 @@ public static Bucket createBucket(String bucketName) throws IOException {
   }
 
   public static Blob uploadObject(String bucketName, String objectName, String filePath)
-    throws IOException, URISyntaxException {
+      throws IOException, URISyntaxException {
     return getStorageService().create(
-      BlobInfo.newBuilder(BlobId.of(bucketName, objectName)).build(),
-      Files.readAllBytes(Paths.get(StorageClient.class.getResource("/" + filePath).toURI())));
+        BlobInfo.newBuilder(BlobId.of(bucketName, objectName)).build(),
+        Files.readAllBytes(Paths.get(StorageClient.class.getResource("/" + filePath).toURI())));
```

Comment:
Same here , undo the changes

Reply:
Done

Reply:
change is not seen it is still seen as new line

Reply:
Changes has been done , as per the review comment